### PR TITLE
Correcting github workflow build script

### PR DESCRIPTION
**Pull request by hamad12a**
I am unaware of how to ignore the rest commits except this one [302aca0](https://github.com/SchoofsKelvin/vscode-sshfs/pull/374/commits/302aca07c2dffd441bb7b52683e61924a1b062e1)

### CODE REVIEW

File: .github/workflows/build-extension.yml
Comment (hamad12a) [inline .github/workflows/build-extension.yml:1]: These are the intended changes that should take place!

Comment (hamad12a) [inline .github/workflows/build-extension.yml:1]: These are the intended changes that should take place instead of the rest useless commits

**Comment by SchoofsKelvin** [comment]
What exactly is this pull request supposed to accomplish? It looks to me like you want it to make a release (those found [here](https://github.com/SchoofsKelvin/vscode-sshfs/releases)) on every commit instead of only when there's a `tag_version`, i.e. if the workflow got started by [pushing a semver tag](https://github.com/SchoofsKelvin/event-utilities/blob/d60695b4be66794fbe0be55ddfc576e8819a999e/src/index.ts#L24-L28).

**Comment by hamad12a** [comment]
You are right! I missed the point of how to create a release when there is a tag_version. I will cancel this pull request.

---- Kelvin Schoofs wrote ----


What exactly is this pull request supposed to accomplish? It looks to me like you want it to make a release (those found here<https://github.com/SchoofsKelvin/vscode-sshfs/releases>) on every commit instead of only when there's a tag_version, i.e. if the workflow got started by pushing a semver tag<https://github.com/SchoofsKelvin/event-utilities/blob/d60695b4be66794fbe0be55ddfc576e8819a999e/src/index.ts#L24-L28>.

**Pull request closed without merging**